packages/assistify-ai/server/lib/SmartiAdapter.js

```diff
@@ -75,7 +75,7 @@ export class SmartiAdapter {
 			const conversation = SmartiProxy.propagateToSmarti(verbs.get,
 				`legacy/rocket.chat?channel_id=${ message.rid }`, null, (error) => {
 					// 404 is expected if no mapping exists
-					if (error.response.statusCode === 404) {
+					if (!error.response || error.response.statusCode === 404) {
```

Comment:
when does this occur?

Reply:
When Smarti is offline or not reachable, `error.response.statusCode` will cause an error because `response is undefined` and therefor the whole method will crash and I can't mark the room as `outOfSync`